server_cmds.go

```diff
@@ -15,129 +15,149 @@ type ServerMethods struct {
-	FlagPassword                           bool    `ms:"virtualserver_flag_password"`
-	LogChannel                             bool    `ms:"virtualserver_log_channel"`
-	LogClient                              bool    `ms:"virtualserver_log_client"`
-	LogFileTransfer                        bool    `ms:"virtualserver_log_filetransfer"`
-	LogPermissions                         bool    `ms:"virtualserver_log_permissions"`
-	LogQuery                               bool    `ms:"virtualserver_log_client"`
-	LogServer                              bool    `ms:"virtualserver_log_server"`
-	WebListEnabled                         bool    `ms:"virtualserver_web_list_enabled"`
-	PrioritySpeakerDimmModificator         float32 `ms:"virtualserver_priority_speaker_dimm_modificator"`
-	BandwidthReceivedLastMinuteTotal       int     `ms:"virtualserver_bandwidth_received_last_minute_total"`
-	BandwidthReceivedLastSecondTotal       int     `ms:"virtualserver_bandwidth_received_last_second_total"`
-	BandwidthSentLastMinuteTotal           int     `ms:"virtualserver_bandwidth_sent_last_minute_total"`
-	BandwidthSentLastSecondTotal           int     `ms:"virtualserver_bandwidth_sent_last_second_total"`
-	ChannelTempDeleteDelayDefault          int     `ms:"virtualserver_channel_temp_delete_delay_default"`
-	HostBannerMode                         int     `ms:"virtualserver_hostbanner_mode"`
-	IconID                                 int     `ms:"virtualserver_icon_id"`
-	MinAndroidVersion                      int     `ms:"virtualserver_min_android_version"`
-	MinClientVersion                       int     `ms:"virtualserver_min_client_version"`
-	MiniOSVersion                          int     `ms:"virtualserver_min_ios_version"`
-	ReservedSlots                          int     `ms:"virtualserver_reserved_slots"`
-	TotalPing                              float32 `ms:"virtualserver_total_ping"`
+	NamePhonetic                           string  `ms:"virtualserver_name_phonetic"`
+	IP                                     string  `ms:"virtualserver_ip"`
+	ServerNickname                         string  `ms:"virtualserver_nickname"`
+	Status                                 string  `ms:"virtualserver_status"`
+	Created                                uint64  `ms:"virtualserver_created"`
 	MaxDownloadTotalBandwidth              uint64  `ms:"virtualserver_max_download_total_bandwidth"`
 	MaxUploadTotalBandwidth                uint64  `ms:"virtualserver_max_upload_total_bandwidth"`
+	ClientConnections                      uint64  `ms:"virtualserver_client_connections"`
+	QueryClientConnections                 uint64  `ms:"virtualserver_query_client_connections"`
+	VirtualServerDownloadQuota             uint64  `ms:"virtualserver_download_quota"`
+	VirtualServerUploadQuota               uint64  `ms:"virtualserver_upload_quota"`
+	FileTransferBandwidthSent              uint64  `ms:"connection_filetransfer_bandwidth_sent"`
```

Comment:
Are these from a different struct or should we put them in a `Connection` sub struct?